backend/data/processors/conditions_preprocessing.py

```diff
@@ -424,6 +424,56 @@ def map_word_to_program_type(processed, regex_word, type):
+    - ... enrolment in an Education program ...
+
+    - ... Enrolment with consent ...                                          To me we should just get rid of it or it's a warning?
+
+    - Enrolment in a School of the Arts && Media honours program
+    - Enrolment in a School of Humanities && Languages honours program
+    - Enrolment in a School of Social Sciences, Asian Studies || European Studies honours program
+    - Enrolment in a Creative Writing honours program
+    - ... enrolment in a Global Diploma program
+    - Enrolment in a Construction Management && Property undergraduate program || minor
+    - ... Enrolment in a Politics, Philosophy && Economics program
+    - ... enrolment in a Social Work program ...
+    - ... enrolment in a Social Science || Social Research && Policy program ...
+    - ... Enrolment in a single || dual award Media program ...
+    - ... enrolment in a single || double degree Media program
+    - Enrolment in a single || double Music (Honours) program
+    - ... Enrolment in a single || double Music (Honours) program
+    - ... Enrolment in a Music program ...
+
+    - Enrolment in Media, Culture && Technology honours program
+    - Enrolment in Theatre && Performance Studies honours program
+    - Enrolment in Environmental Humanities honours program
+    - ... enrolment in program 4053 || 4054
+    - ... enrolment in program 4076
+    - Enrolment in CDF Program (4462)
+    - Enrolment in Business (Honours) Program 4512
+    - Enrolment in CDF Program
+    - Enrolment in CDF Program (4461)
+    - Enrolment in program 4508 Music (Honours)
+    - Enrolment in Physics Honours
+    - Enrolment in 4529 Social Science (Honours)
+    - Enrolment in 3959 Data Science program ...                                                only once, but data science is close to CS
+    - Enrolment in program 3831 Science (Medicine) Honours
+    - ... Enrolment in 7006 UNSW Global Diploma in Architecture Program                         Need to be careful about this one, sometimes there is stuff before the requirement, sometimes not
+    - Enrolment in 7005 UNSW Global Diploma in Media && Communication Program
+
+    - Enrolment in the (Education/Criminology/Film Studies/English/Dance Studies/History/Philosophy/Asian Studies/Chinese Studies/French Studies/Spanish Studies/Japanese Studies/Korean Studies/Linguistics/German Studies/Development Studies/European Studies/International Relations/Politics/Sociology && Anthropology/Politics && International Relations/Global Development/Media) honours program
+    - Enrolment in the Social Research && Policy honours program
+
+    - ... Must pass for WIL modules prior to enrolment ...                    Maybe this is a warning though, not a specialisation?
+
+    - LAWS2277 seems to have been filtered wrong: [LAWS1160 || JURD7160 (One of these courses must be prior to || concurrently with enrolment in)]
+    """
```

Comment:
Are these just notes for things to implement later?

Reply:
Yep just notes for common things to do with specialisations that I'll deal with in the next few days before merging